src/Tribe/REST/V1/Service_Provider.php

```diff
@@ -231,49 +253,111 @@ protected function hooks() {
 		add_filter( 'tribe_rest_event_data', tribe_callback( 'tickets.rest-v1.flags', 'flag_ticketed_event' ), 10, 2 );
 	}
 
+	/**
+	 * Register the single attendee endpoint.
+	 *
+	 * @return Tribe__Tickets__REST__V1__Endpoints__Single_Attendee
+	 */
 	protected function register_single_attendee_endpoint() {
 		/** @var Tribe__Tickets__REST__V1__Endpoints__Single_Attendee $endpoint */
 		$endpoint = tribe( 'tickets.rest-v1.endpoints.attendees-single' );
 
-		register_rest_route( $this->namespace, '/attendees/(?P<id>\\d+)', array(
-			'methods'             => WP_REST_Server::READABLE,
-			'args'                => $endpoint->READ_args(),
-			'callback'            => array( $endpoint, 'get' ),
-			'permission_callback' => '__return_true',
-		) );
-
-		register_rest_route( $this->namespace, '/attendees/', [
-			'methods'             => WP_REST_Server::CREATABLE,
-			'args'                => $endpoint->CREATE_args(),
-			'callback'            => [ $endpoint, 'create' ],
-			'permission_callback' => [ $endpoint, 'validate_user_permission' ],
-		] );
-
-		register_rest_route( $this->namespace, '/attendees/(?P<id>\\d+)', [
-			'methods'             => WP_REST_Server::EDITABLE,
-			'args'                => $endpoint->EDIT_args(),
-			'callback'            => [ $endpoint, 'update' ],
-			'permission_callback' => [ $endpoint, 'validate_user_permission' ],
-		] );
+		register_rest_route(
+			$this->namespace,
+			'/attendees/(?P<id>\\d+)',
+			[
+				'methods'             => WP_REST_Server::READABLE,
+				'args'                => $endpoint->READ_args(),
+				'callback'            => [ $endpoint, 'get' ],
+				'permission_callback' => '__return_true',
+			]
+		);
+
+		register_rest_route(
+			$this->namespace,
+			'/attendees/',
+			[
+				'methods'             => WP_REST_Server::CREATABLE,
+				'args'                => $endpoint->CREATE_args(),
+				'callback'            => [ $endpoint, 'create' ],
+				'permission_callback' => [ $endpoint, 'validate_user_permission' ],
+			]
+		);
+
+		register_rest_route(
+			$this->namespace,
+			'/attendees/(?P<id>\\d+)',
+			[
+				'methods'             => WP_REST_Server::EDITABLE,
+				'args'                => $endpoint->EDIT_args(),
+				'callback'            => [ $endpoint, 'update' ],
+				'permission_callback' => [ $endpoint, 'validate_user_permission' ],
+			]
+		);
 
 		tribe( 'tickets.rest-v1.endpoints.documentation' )->register_documentation_provider( '/attendees/{id}', $endpoint );
 
 		return $endpoint;
 	}
 
+	/**
+	 * Register the attendees archive endpoint.
+	 *
+	 * @return Tribe__Tickets__REST__V1__Endpoints__Attendee_Archive
+	 */
```

Comment:
```suggestion
	/**
	 * Register the attendees archive endpoint.
	 *
	 * @since TBD
	 *
	 * @return Tribe__Tickets__REST__V1__Endpoints__Attendee_Archive
	 */
```